src/Components/NoSytemsAlert/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Alert,
+  AlertActionCloseButton,
+  ClipboardCopy,
+  Form,
+  FormGroup,
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  Title,
+} from '@patternfly/react-core';
+import { CopyIcon } from '@patternfly/react-icons';
+
+function NoSystemsAlert({ handleClose }) {
+  return (
```

Comment:
Extra `return` statement.

Reply:
Its a function :wink:

Reply:
Aaah, why not making it a const? :smile:

Reply:
:smiling_imp: to confuse the hell out of you all :D Do you want me to make it a const?

Reply:
much appreciated!